pkg/api/http/actors.go

```diff
@@ -70,7 +70,15 @@ func actorInvocationMethodNameFn(r *http.Request) string {
 	return "InvokeActor/" + chi.URLParam(r, actorTypeParam) + "." + chi.URLParam(r, actorIDParam)
 }
 
+func actorInvocationLowCardinalityMethodNameFn(r *http.Request) string {
```

Comment:
I would like us to get away from using "low cardinality" as a concept in Dapr. Let's rename this func to `actorInvocationMethodNameFn` and the above func to `actorInvocationMethodNameWithIDFn`.

Reply:
fixed